docs/developerguide.rst

```diff
@@ -56,3 +56,135 @@ We are using Prettier to format our code. In order to use it before a Pull Reque
    .. code-block:: console
 
       $ prettier --write **/*.js
+
+CERN
+----
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You must run the Minikube cluster with VirtualBox VM so you listen requests from outside the
```

Comment:
What do you think about splitting into bullets? An approximation could be this:
- Be inside the CERN network
- Set port forwarding from your laptop to the VM where REANA runs. The supported way of doing so is using VirtualBox VM driver for Minikube. To set it up you can follow:
    - go to the VirtualBox UI and select
the running VM, then in the *Network* section select *Advanced* > *Port Forwarding* (followed by the screenshot)
    - Add a new entry (in the screenshot ``reana-backend``) to forward all the requests going to the port
443 (default for HTTPS) to the internal port 30443 (where REANA-Server is listening).